web/tests/settings_org.test.js

```diff
@@ -156,7 +155,7 @@ function test_submit_settings_form(override, submit_form) {
     ]);
 
     patched = false;
-    submit_form(ev);
+    submit_form.call($(".save-discard-widget-button.save-button"), ev);
```

Comment:
Some of our existing tests are deeply confused on this point, but in reality, `this` is an `HTMLElement`, never a jQuery object wrapping one. The cleanest way to model this within the existing framework is probably

```suggestion
    submit_form.call({to_$: () => $(".save-discard-widget-button.save-button")}, ev);
```